### PR TITLE
[WIP] switch to Q35

### PR DESCRIPTION
**Pull request by devimc**
switch to q35

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
depends on https://github.com/ciao-project/ciao/pull/1517

File: config/configuration.toml.in [hunk 1]
```diff
@@ -10,6 +10,10 @@ machine_type = "@MACHINETYPE@"
 # trouble running pre-2.15 glibc
 kernel_params = "@KERNELPARAMS@"
 
+# Path to the BIOS.
+# If you want that qemu uses the default BIOS leave this option empty
```
Comment (jodh-intel) [inline config/configuration.toml.in]: Do we need to say anything in this comment about `machine_type`? Will this only work for `q35`? Will it work for any value? *etc*.

File: config.go [hunk 2]
```diff
@@ -134,6 +135,16 @@ func (h hypervisor) image() (string, error) {
 	return resolvePath(p)
 }
 
+func (h hypervisor) bios() (string, error) {
```
Comment (jodh-intel) [inline config.go]: This needs a new test :)

File: config/configuration.toml.in (hunk 1 above)
Comment (grahamwhaley) [reply config/configuration.toml.in]: This looks to me to be independant of machine type and q35 - so I'd drop the mention of q35 or machine type at all from the series - just say it allows a choice of BIOS.

**Comment by devimc** [comment]
depends on https://github.com/containers/virtcontainers/pull/403

**Pull request closed without merging**